readme.txt

```diff
@@ -2,9 +2,9 @@
 * Contributors:      pluginkollektiv
 * Donate link:       https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=TD4AMD2D8EMZW
 * Tags:              analytics, dashboard, pageviews, privacy, statistics, stats, visits, web stats, widget
-* Requires at least: 4.7
-* Tested up to:      6.3
-* Requires PHP:      5.3
+* Requires at least: 5.1
+* Tested up to:      6.4
```

Comment:
Could we already raise that to 6.5?

Reply:
Sure, why not. Works fine on a local 6.5 RC test site and even if there's something wrong with it, we should be able to fix this before release.